openapi-maven-plugin/src/main/java/io/github/kbuntrock/reflection/ReflectionsUtils.java

```diff
@@ -39,20 +37,6 @@ public static ClassLoader getProjectClassLoader() {
 		return ReflectionsUtils.projectClassLoader;
 	}
 
-	public static ConfigurationBuilder createConfigurationBuilder() {
```

Comment:
I like this! the "testMode" boolean is not needed anymore now, I will remove it.